weedcoco/repo/deposit.py

```diff
@@ -123,16 +128,16 @@ def extract(self, dest_dir, version="head"):
             objdir=str(self.object_path), version=version, dstdir=dest_dir
         )
 
-    def validate(self):
+    def validate_image(self):
```

Comment:
I think this was better as validate() in case we want to include other checks which aren't to do with image validation